NEWS.md

```diff
@@ -0,0 +1,111 @@
+ 	"github.com/cosmos/cosmos-sdk/baseapp"
+ 	"github.com/cosmos/cosmos-sdk/server"
+-	"github.com/cosmos/cosmos-sdk/store"
+ 	genutilcli "github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+ 	"github.com/cosmos/cosmos-sdk/x/staking"
+ )
+@@ -87,9 +86,14 @@ func main() {
+ }
+ 
+ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer) abci.Application {
++	pruningOpts, err := server.GetPruningOptionsFromFlags()
++	if err != nil {
++		panic(err)
++	}
++
+ 	return app.NewCommercioNetworkApp(
+ 		logger, db, traceStore, true, invCheckPeriod,
+-		baseapp.SetPruning(store.NewPruningOptionsFromString(viper.GetString("pruning"))),
++		baseapp.SetPruning(pruningOpts),
+ 		baseapp.SetMinGasPrices(viper.GetString(server.FlagMinGasPrices)),
+ 		baseapp.SetHaltHeight(uint64(viper.GetInt(server.FlagHaltHeight))),
+ 	)
+```
+
+### Migrate a node from 0.38.5 to 0.39.0
+
+Note: **do not modify pruning settings with any release prior to `v0.39.0` as that may cause data corruption**.
+The following instructions assume that **pruning settings have not been modified since the node started using 0.38.x. Note: the default pruning setting `syncable` used `KeepEvery:100`.
+
+* The simple upgrade strategy: **perform a full sync of the node from scratch**. Else, follow one of the other strategies.
+
+* If your node had started with using `KeepEvery:1` (e.g. pruning settings `nothing` or `everything`), upgrading to `v0.39.0` is safe.
+
+* Otherwise, do halt block processing with `--halt-height` after committing a height divisible by `KeepEvery` - e.g. at block 147600 with `KeepEvery:100`. The **node must never have processed a height beyond that at any time in its past**. Upgrading to `v0.39.0` is then safe.
+
+* Otherwise, set the `KeepEvery` setting to the same as the previous `KeepEvery` setting (both `<=v0.38.5` and `v0.39.0` default to `KeepEvery:100`). Upgrade to `v0.39.0` is then safe as long as you wait one `KeepEvery` interval plus one `KeepRecent` interval **plus** one pruning `Interval` before changing pruning settings or deleting the last `<=v0.38.5` height (so wait *210* heights with the default configuration).
+
+* Otherwise, make sure the last version persisted with `<=v0.38.5` is never deleted after upgrading to `v0.39.0`, as doing so may cause data loss and data corruption.
+
+## Regression in the signature verification when multiple transactions in the same block are sent from the same account
+
+When multiple transactions in the same block are sent (and correctly signed) by the same account, chances are that some of them could be rejected and the error `unauthorized: signature verification failed` would be returned due to the account's sequence (*nonce*) getting stuck and not being incremented by the ante handler. This behaviour was [a regression](https://github.com/cosmos/cosmos-sdk/issues/6287) introduced in the `v0.38` release series and did not occur in the `v0.37` release series.
```

Comment:
Could we be more explicit in this section that the regression has been fixed in 0.39?

Reply:
Good point, please see https://github.com/cosmos/cosmos-sdk/pull/6707/commits/c0fe624b1c6f1d44a8c0bdefb299d5afdfc29dbf